### DISCUSSION TITLE
PWA no longer working

### DISCUSSION THREAD

**Discussion by arnipallg** [filips123/PWAsForFirefox#160] [Q&A] [answered]
Using Firefox 101.0 and version 1.4.0, Google Chat stopped working as an PWA stopped working, launching the PWA now only opens a blank Firefox window.
I have tried restarting Firefox, reinstalling the extension, reinstalling the PWA, all to no avail. How did I mess this up?

**Comment by filips123** [comment, marked as answer]
You need to use 1.4.2 because of some bugs (see #154, #155, #157). However, it is not yet released on the addon store, so you will need to manually download it from [here](https://github.com/filips123/PWAsForFirefox/releases/tag/v1.4.2) (then, the extension will say it's outdated but this is fine for now).

**Reply by arnipallg** [reply]
Thank you kindly for a swift resolution.
